src/Common/src/TypeSystem/IL/Stubs/DelegateMarshallingMethodThunk.Mangling.cs

```diff
@@ -23,13 +24,17 @@ string IPrefixMangledType.Prefix
         {
             get
             {
-                if (IsOpenStaticDelegate)
+                switch (Kind)
```

Comment:
Make sense. I'll do that for the other Interop NameMangling classes(eg. struct marshalling) too.